libcudacxx/include/cuda/__memory_resource/get_memory_resource.h

```diff
@@ -21,16 +21,17 @@
 #  pragma system_header
 #endif // no system header
 
-#include <cuda/__memory_resource/properties.h>
-#include <cuda/std/__execution/env.h>
-#include <cuda/std/__type_traits/is_same.h>
+#if defined(LIBCUDACXX_ENABLE_EXPERIMENTAL_MEMORY_RESOURCE)
```

Comment:
question: I believe there should be nothing experimental about `get_memory_resource` query, which is one of the reasons allowing us to move it from `cudax::` to `cuda::`. Is there any way we could move the query outside of `LIBCUDACXX_ENABLE_EXPERIMENTAL_MEMORY_RESOURCE`? CUB doesn't control the setup on user end. With `get_memory_resource` query under the feature macro, it'll be hard to expose env-based overload that queries a memory resource.

Reply:
As discussed in person, we provide an internal `__get_memory_resource_t` until we are droping the experimental requirement on resources